<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>643</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Streaming AI with Retrigger - Configuration</string>
  </property>
  <widget class="QComboBox" name="cmbDevice">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDevice">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>91</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpAISetting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Streaming AI settings</string>
   </property>
   <widget class="QComboBox" name="cmbChannelCount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannelCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Channel count:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblRateUnit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbValueRange">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbChannelStart">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtClockRatePerChan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblValueRange">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Value range:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblClockRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Clock rate:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannelStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Channel start:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelStart</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSectionLength">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10240</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblSectionLength">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Section Length:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtCycles">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblCycles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Cycles:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpTriggerSetting">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Trigger settings</string>
   </property>
   <widget class="QComboBox" name="cmbTriggerEdge">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>47</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerEdge">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>45</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Edge:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerDelayCount">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>76</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Delay count:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerLevelUnit">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>102</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>21</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>V</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbTriggerSource">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>18</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTriggerLevel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>103</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerLevel">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>103</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Trigger level:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDelayCount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>75</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerSource">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>18</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Source:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelStart</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpTriggerSetting_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Trigger 1 settings</string>
   </property>
   <widget class="QComboBox" name="cmbTriggerEdge_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerEdge_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Edge:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerDelayCount_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>79</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Delay count:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerLevelUnit_2">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>108</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>21</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>V</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbTriggerSource_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>18</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTriggerLevel_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>109</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerLevel_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>109</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Trigger level:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDelayCount_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>79</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTriggerSource_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>18</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Source:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelStart</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblProfileName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>91</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Profile:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtProfilePath">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cmbDevice</tabstop>
  <tabstop>cmbChannelStart</tabstop>
  <tabstop>cmbChannelCount</tabstop>
  <tabstop>cmbValueRange</tabstop>
  <tabstop>edtClockRatePerChan</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
